apps/explorer/components/Contract/index.tsx

```diff
@@ -157,14 +167,14 @@ export function Contract({ contract, rates, renewedFrom, renewedTo }: Props) {
         {
           label: 'renter payout: remaining renter allowance',
           initials: 'r',
-          sc: new BigNumber(renterPayoutValid.value),
-          hash: renterPayoutValid.output_id,
+          sc: renterPayoutValid && new BigNumber(renterPayoutValid.value),
+          hash: renterPayoutValid?.address,
```

Comment:
output id and address are not the same, if we are changing what is displayed just make sure this is labeled correctly.

Reply:
Also leaving this open before I submit this PR: these are different but unlabeled. Not sure what the best course of action is on this one.

Reply:
Looks like throughout the PR we are swapping output IDs for addresses, lets ask whether we should add output ID to the API and continue to provide both, or change these lists to display these things as addresses only. The EntityListItem has a `type` we can set to address. Elsewhere such as on the Transaction page you have already migrated the EntityList renders OutputListItems which use and display both the output ID and the address (which navigates to the address page).
<img width="676" alt="Screenshot 2024-10-25 at 9 00 44 AM" src="https://github.com/user-attachments/assets/64ac2acf-dd08-4f92-859d-95cfc72a13f2">
<img width="659" alt="Screenshot 2024-10-25 at 9 00 09 AM" src="https://github.com/user-attachments/assets/d537a1b1-e69d-4245-934c-f6505ee39634">